quickwit-cluster/src/lib.rs

```diff
@@ -78,15 +78,15 @@ pub async fn start_cluster_service(
     let member = Member::new(
         quickwit_config.node_id.clone(),
         unix_timestamp(),
-        quickwit_config.gossip_public_addr().await?,
+        quickwit_config.gossip_advertise_addr().await?,
```

Comment:
I personally preferred public actually

Reply:
Cassandra uses [broadcast_address](https://docs.datastax.com/en/cassandra-oss/3.x/cassandra/configuration/configCassandra_yaml.html#configCassandra_yaml__broadcast_address) 

Docs description is clear.

I'm both ok with advertise / public, I have a slight preference for advertise though.

Reply:
I'm going to stick with `advertise_address`, which is also used for the Kube API server. My grief with `public_ip` is that the address usually used for inter-node communication is actually private (`192.168.`, `172.16.`).

Reply:
ok